examples/realitytabs.html

```diff
@@ -1036,7 +1037,9 @@
           code === 27 || // esc
           code === 91 // win
         ) {
-          // nothing
+          if (code === 16) { // shift
+            isShifted = !isShifted;
```

Comment:
This might work functionally, but it would be good to actually render blue shift key overtop or something to signal to the user the key is "on".